pkg/services/requests.go

```diff
@@ -18,7 +18,15 @@ import (
 )
 
 const (
-	requestWaitingForApprovalStatus = "Pending Approval"
+	AccessRequestInitStatus               = "Initializing"
+	AccessRequestPendingStatus            = "Pending"
+	AccessRequestGrantingStatus           = "Granting"
+	AccessRequestRejectedStatus           = "Rejected"
+	AccessRequestActiveStatus             = "Active"
+	AccessRequestRevokingStatus           = "Revoking"
+	AccessRequestRevokedStatus            = "Revoked"
+	AccessRequestFailedStatus             = "Failed"
+	AccessRequestWaitingForApprovalStatus = "Pending Approval"
```

Comment:
Pending and Pending Approval are the same one
Actual value returned is "Pending"
if you want to show user "Pending approval" you need to ma it everywhere
But not sure I would go this way

Reply:
I do it with the `ColoredStatus` function